WTWSMS/history/titles/c_aswan.txt

```diff
@@ -1,4 +1,11 @@
 465.1.1={
 	holder=215016
 	liege = "k_egypt"
+}
+560.1.1={
+	holder=1310011
+	liege = "k_egypt"
+}
+610.1.1={
+    holder=0
```

Comment:
Counties shouldn't have holder = 0.

Reply:
The character doesn't die 'till 610 but he stops holding it at 593. Do I give it to the Exarch of Ravenna?

Reply:
Solved